tegel/src/components/datetime/datetime.tsx

```diff
@@ -54,7 +54,24 @@ export class Datetime {
     bubbles: true,
     cancelable: true,
   })
-  customChange: EventEmitter;
+  sddsChange: EventEmitter;
+
+  /** Blur event for the datetime */
+  @Event({
+    composed: true,
+    bubbles: true,
+    cancelable: true,
+  })
+  sddsBlur: EventEmitter;
```

Comment:
```suggestion
  sddsBlur: EventEmitter<FocusEvent>;
```
Just to be extra clear to the user we could type this as a FocusEvent